src/com/company/Match.java

```diff
@@ -0,0 +1,142 @@
+package com.company;
+
+import com.company.enums.MatchType;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import static com.company.InputValidator.validateMatchType;
+
+public class Match {
+    int numberOfOvers;
+    Team team1 = new Team();
+    Team team2 = new Team();
+
+    Match(String teamName1, String teamName2){
+        team1.setTeamName(teamName1);
+        team2.setTeamName(teamName2);
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Starting A New Match");
+        System.out.println("Which Format Match you want : FIVE_OVER / T20 / FIFTY_OVER");
+        MatchType applyingMatchType;
+        String matchTypeByUser = sc.next();
+        String defaultMatchType = "FIVE_OVER";
+        boolean validateInput = validateMatchType(matchTypeByUser);
+        if(!validateInput){
+            System.out.println("You Provide inappropriate input, So by default making match of 5 Overs");
+            applyingMatchType = MatchType.valueOf(defaultMatchType);
+        }
+        else{
+            // convert String to enum
+            applyingMatchType = MatchType.valueOf(matchTypeByUser);
+        }
+        numberOfOvers = applyingMatchType.getOverInThisType();
+        startMatch();
+    }
+
+    private void startMatch(){
+        int winnerOfToss = performToss();
+        performInningSchedule(winnerOfToss);
+        showFinalResult();
+    }
+
+    private void performInningSchedule(int winnerOfToss){
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Would You like to Batting or Bowling");
```

Comment:
method is too long. pls break down into subMethods.

As discussed, no method should be above 20 lines.